jsoup/src/test/java/com/baeldung/jsoup/JsoupParserTest.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.jsoup;
+
+import java.io.IOException;
+import static junit.framework.Assert.assertEquals;
```

Comment:
The junit.framework package is deprecated in JUnit 4+.
Change these static imports to org.junit.Assert.assertEquals and org.junit.Assert.assertTrue